appConfig.js

```diff
@@ -1,4 +1,4 @@
-export default {
+module.exports = {
```

Comment:
Was this for tests?

Reply:
oh that was just because I didn't want to have `import config from "appconfig.js"` and then destructure after. w cjs you can destructure inline on the import. i can change it back.